lib/Digitick/Sepa/GroupHeader.php

```diff
@@ -87,12 +87,12 @@ class GroupHeader
     /**
      * @param string $messageIdentification Maximum length: 35. Reference Number of the bulk.
      *                                      Part of the duplication check (unique daily reference).
-     *                                      The first 8 or 11 characters of <Msgld> must match the BIC of the
-     *                                      Instructing Agent. The rest of the field can be freely defined.
+     *                                      The HCT domestic default <Msgld> contains the prefix 'HUF-'
+     *                                      followed by the date+time using the format 'ymdHis'.
      * @param string $initiatingPartyName
      * @param boolean $isTest
      */
-    public function __construct($messageIdentification, $initiatingPartyName, $isTest = false)
+    public function __construct($messageIdentification = 'HUF-'.date('ymdHis'), $initiatingPartyName, $isTest = false)
```

Comment:
beside that you can not use a function as default value in a constructor Its only some sugar when creating the GroupHeader, this can be moved to the facade or a factory ...propably we can create a second package which handles HUF as separate solution.

Reply:
This change one can be dropped, as long as the written requirement about the content of $messageIdentification is not enforced, the user can provide the HUF-YYMMDDHHiiss during class instantiation.

Reply:
And sorry that I explicitly named the BC breaks again you already mentioned in your PR I overlooked them in my initial reading